File: app/src/main/java/com/tronography/pixelweather/utils/IconUrlUtils.java
package com.tronography.pixelweather.utils;


public class IconUrlUtils {

    private static final String ICON_BASE_URL = "http://openweathermap.org/img/w/";
    private static final String PNG = ".png";

    public static String getIconUrl(String icon){
        return new StringBuilder()
                .append(ICON_BASE_URL)
                .append(icon)
                .append(PNG)
                .toString();
    }

}

File: app/src/main/java/com/tronography/pixelweather/utils/IconUrlUtils.kt
package com.tronography.pixelweather.utils


object IconUrlUtils {

    private val ICON_BASE_URL = "http://openweathermap.org/img/w/"
    private val PNG = ".png"

    fun getIconUrl(icon: String): String {
        return StringBuilder()
                .append(ICON_BASE_URL)
                .append(icon)
                .append(PNG)
                .toString()
    }

}
